dev/PushNotifications/PushNotificationsLongRunningTask.ProxyStub/PushNotificationsLRP.idl

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+import "oaidl.idl";
+import "ocidl.idl";
+
+#include "../PushNotifications-Constants.h"
+
+[object]
+[uuid(60FC21B2-B396-4D49-94F0-7555869FB93C)]
+[pointer_default(unique)]
+interface IWpnLrpPlatform : IUnknown
```

Comment:
Nit: 
In general, avoid abbreviations in names. 
For example, filename could be NotificationsLongRunningProcess. LAter this LRP would be applicable to other scenarios such as toast activations as well.

IWpnLrpPlatform. Lets call it INotificationsLongRunningPlatform.
Same for other names below.